configure

```diff
@@ -3733,14 +3728,8 @@ cat confdefs.h - <<_ACEOF >conftest.$ac_ext
 
 /* Override any GCC internal prototype to avoid an error.
    Use char because int might match the return type of a GCC
-   builtin and then its argument prototype would still apply.
-   The 'extern "C"' is for builds by C++ compilers;
-   although this is not generally supported in C code supporting it here
-   has little cost and some practical benefit (sr 110532).  */
-#ifdef __cplusplus
-extern "C"
-#endif
-char mpc_init2 (void);
+   builtin and then its argument prototype would still apply.  */
+char mpc_init2 ();
```

Comment:
Ditto?
And same for lots of other changes following below.

Reply:
https://github.com/riscv-collab/riscv-gnu-toolchain/pull/1559/files#r1795721519